archunit-junit/junit5/engine/src/main/java/com/tngtech/archunit/junit/ConditionEvaluator.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2014-2022 TNG Technology Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.tngtech.archunit.junit;
+
+import org.junit.jupiter.api.extension.ConditionEvaluationResult;
+import org.junit.jupiter.api.extension.ExecutionCondition;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.platform.commons.JUnitException;
+
+class ConditionEvaluator {
+
+    ConditionEvaluationResult evaluate(ArchUnitEngineExecutionContext context, ExtensionContext extensionContext) {
```

Comment:
This class is analogous to `org.junit.jupiter.engine.execution.ConditionEvaluator`